src/routes/api/cryptor-migrations/migration-router.ts

```diff
@@ -0,0 +1,66 @@
+import { Request, Response } from "express";
+import { Installation } from "models/installation";
+
+const MAX_BATCH_SIZE = 10_000;
+const DEFAULT_BATCH_SIZE = 10;
+const RESPONSE_BATCH_SIZE = 1000;
```

Comment:
bad naming :( keen for suggestion. See bellow. Indicate how often do we tell the curl client that xxx has been migrated.